src/containers/TokenPageContainer/TokenPageContainer.jsx

```diff
@@ -440,7 +440,7 @@ const TokenPageContainer = () => {
     if (offerData?.nftContract?.tokens.length > 0) {
       setMarketplaceListings(
         offerData?.nftContract?.tokens
-          ?.find((token) => Number(token?.tokenId) === Number(tokenId))
+          ?.find((token) => BigInt(token?.tokenId) === BigInt(tokenId))
           ?.marketplaceListings.sort((a, b) => b.id - a.id)
```

Comment:
you have to update the sorting function 
https://stackoverflow.com/questions/65435403/array-sorting-is-broken-with-bigint-in-js

Reply:
I'll use this then

`const mixed = [4n, 6, -12n, 10, 4, 0, 0n];
// ↪  [4n, 6, -12n, 10, 4, 0, 0n]

mixed.sort();
// ↪ [-12n, 0, 0n, 10, 4n, 4, 6]

mixed.sort((a, b) => a - b);
// ne fonctionnera pas car la soustraction ne
// fonctionne pas sur des types mixtes
// TypeError: can't convert BigInt value to Number value

// On peut trier avec un comparateur numérique approprié
mixed.sort((a, b) => (a < b ? -1 : a > b ? 1 : 0));
// ↪ [ -12n, 0, 0n, 4n, 4, 6, 10 ]
`

https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/BigInt#m%C3%A9thodes_dinstance

Reply:
we don't even need to change the sorting since we instance tokenId as a BigInt only above, the sorting still uses marketplacesListing id which is a number

Reply:
oh yes, my bad, I have not checked correctly that sorting is on `marketplaceListings.id`, not `nftContract.tokens.tokenId`
so you don't have to fix on sorting